libs/payments/ui/src/lib/server/purchase-details.ftl

```diff
@@ -0,0 +1,19 @@
+## Component - PurchaseDetails
+
+## Subscription upgrade plan details - shared by multiple components, including plan details and payment form
+## $amount (Number) - The amount billed. It will be formatted as currency.
+
+# $intervalCount (Number) - The interval between payments, in days.
```

Comment:
What is `$intervalCount` referring to? I don't see references to that variable used in this file.

Reply:
tl;dr - No longer necessary, copy/paste mistake. 

I had to remind myself what was happening here.

With SP3.0 we are moving towards supporting only a specific set of intervals, so the intervalCount is no longer necessary.